src/input/input.directive.ts

```diff
@@ -1,5 +1,15 @@
 import { Directive, HostBinding } from "@angular/core";
-
+/**
+ * A convinence directive for applying styling to a input element.
```

Comment:
There's a typo in all documentation that has `A convinence directive`. 

I will accept this and change the rest after it goes through

Reply:
Ok I'll change